src/Module.h

```diff
@@ -1,8 +1,8 @@
+#include "TypeDef.h"
+#include "Hal.h"
```

Comment:
Why is this moved before the guard ifdef?

Reply:
I don't remember much about it, but maybe it's because some define were not found, or maybe it's completely by mistake. I remember at least one case like that but I don't know where. I'll try again.